ci-operator/step-registry/baremetal/lab/post/baremetal-lab-post-chain.yaml

```diff
@@ -2,6 +2,7 @@ chain:
   as: baremetal-lab-post
   steps:
     - chain: gather
+    - ref: baremetal-lab-add-nodes
```

Comment:
we should add  this as first step of the `test:` stanza, before running the E2Es. We should do this at the testconfig level